solution/ui/regulations/.eslintrc.json

```diff
@@ -1,15 +1,15 @@
 {
     "env": {
         "browser": true,
-        "es2021": true
+        "es2020": true
```

Comment:
moving backwards?

Reply:
I can probably change this back.  I'm having inconsistent ESLint behavior after we reorganized the repository and this was a suggestion from a stackoverflow article that worked for me.  But I just switched it back this morning and things are looking okay... so I'll revert this to es2021.